frontend_vue/src/components/ui/IncidentDetails.vue

```diff
@@ -1,80 +1,145 @@
+          <li class="flex flex-col gap-2">
+            <span class="text-xs text-red-100">Date</span>
+            <span class="font-semibold"
+              >{{
+                convertUnixTimeStampToDate(props.hitAlert.time_of_hit)
+              }} </span
+            >
+          </li>
+          <li class="flex flex-col gap-2">
+            <span class="text-xs text-red-100">IP</span>
+            <span class="font-semibold">{{ props.hitAlert.geo_info.ip }}</span>
+          </li>
+          <li class="flex flex-col gap-2">
+            <span class="text-xs text-red-100">Channel</span>
+            <span class="font-semibold">{{ props.hitAlert.src_ip }}</span>
+          </li>
+        </ul>
+      </section>
+      <!-- Details -->
+      <section class="grid md:grid-cols-[auto_1fr] gap-32 mt-32 pl-8">
+        <template
+          v-for="(val, key) in formatAlertObject"
+          :key="key"
+        >
+          <h3 class="text-grey-500">
+            {{ key }}
+          </h3>
+          <ul
+            class="flex flex-col gap-16 pb-16 [&:not(:last-child)]:border-b md:ml-32 border-grey-100"
           >
-          <span>AS3320 Deutsche Telekom AG</span>
-        </li>
-      </ul>
-      <h3 class="text-grey-500">Basic info</h3>
-      <ul class="flex flex-col gap-16 pb-16-b md:ml-32">
-        <li>
-          <span class="block text-xs uppercase text-grey-500">User Agent</span>
-          <p>
-            Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36
-            (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36
-          </p>
-        </li>
-        <li>
-          <span class="block text-xs uppercase text-grey-500"
-            >request headers</span
-          >
-          <code class="text-grey-800">
-            {'Host': 'canarytokens.com', 'X-Real-Ip': '84.138.198.174',
-            'X-Forwarded-For': '84.138.198.174', 'X-Forwarded-Host':
-            'canarytokens.org', 'Connection': 'close',
-            'Upgrade-Insecure-Requests': '1', 'User-Agent': 'Mozilla/5.0
-            (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like
-            Gecko) Chrome/122.0.0.0 Safari/537.36', 'Accept':
-            'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7',
-            'Accept-Encoding': 'gzip, deflate', 'Accept-Language':
-            'en-GB,en-US;q=0.9,en;q=0.8,zh-CN;q=0.7,zh;q=0.6,it;q=0.5'}
-          </code>
-        </li>
-      </ul>
-    </section>
+            <template v-if="isNotEmptyObject(val)">
+              <li
+                v-for="(subval, subkey) in val"
+                :key="subkey"
+                class="break-words"
+              >
+                <span class="block text-xs uppercase text-grey-500">{{
+                  subkey
+                }}</span>
+                <span :class="addClassToBoolean(val)">{{ subval }}</span>
```

Comment:
`addClassToBoolean` is a weird name for changing the text colour?

Reply:
yeah you're right, I will change it  🥲